common/src/main/java/org/apache/gravitino/dto/authorization/RoleDTO.java

```diff
@@ -93,6 +94,10 @@ public Map<String, String> properties() {
    */
   @Override
   public List<SecurableObject> securableObjects() {
+    if (securableObjects == null) {
+      return Collections.emptyList();
+    }
```

Comment:
Will we return a null `securableObjects` here?

Reply:
Legacy code. Removed.